docs/power-to-energy.md

```diff
@@ -0,0 +1,153 @@
+# Power-to-Energy
+
+`power-to-energy` is a generic plugin for calculating enegy consumption (kWh) from wattage and duration.
+
+You provide the wattage value and the duration.
+
+For example, you could add `wattage` and `duration`. The `energy-consumption` would then be added to every observation in your input array as the calculated value of `wattage` and `duration`.
+
+## Parameters
+
+### Plugin config
+
+Three parameters are required in config: `input-parameter`, `duration` and `output-parameter`.
+
+- `input-parameter`: a string matching an existing key in the `inputs` array.
+- `duration`: the value to calculate `energy-consumption`.
+- `output-parameter`: a string defining the name to use to add the result of the calculation of the input parameters to the output array.
+
+### Plugin parameter metadata
+
+The `parameter-metadata` section contains information about `description`, `unit` and `aggregation-method`
+of the parameters of the inputs and outputs
+
+- `inputs`: describe parameters of the `input-parameter` of the config. Each parameter has:
+
+  - `description`: description of the parameter
+  - `unit`: unit of the parameter
+  - `aggregation-method`: aggregation method object of the parameter
+    - `time`: this value is used for `horizontal` aggregation. It can be of the following values: `sum`, `avg`, `copy`, or `none`.
+    - `component`: this value is used for `vertical` aggregation. It can be of the following values: `sum`, `avg`, `copy`, or `none`.
+
+- `outputs`: describe parameters of the `output-parameter` of the config. Each parameter has:
+  - `description`: description of the parameter
+  - `unit`: unit of the parameter
+  - `aggregation-method`: aggregation method object of the parameter
+    - `time`: this value is used for `horizontal` aggregation. It can be of the following values: `sum`, `avg`, `copy`, or `none`.
+    - `component`: this value is used for `vertical` aggregation. It can be of the following values: `sum`, `avg`, `copy`, or `none`.
+
+### Mapping
+
+The `mapping` block is an optional block. It is added in the plugin section and allows the plugin to receive a parameter from the input with a different name than the one the plugin uses for data manipulation. The parameter with the mapped name will not appear in the outputs. It also maps the output parameter of the plugin. The structure of the `mapping` block is:
+
+```yaml
+power-to-energy:
+  method: ConvertPowerToEnergy
+  path: 'builtin'
```

Comment:
This is community-plugins, not builtin. Please recheck/update this documents (maybe I could not other points which you should update...)